mxd/3.2_MXD_tutorial2_Access_policies.md

```diff
@@ -0,0 +1,129 @@
+
+Bob will once again be the data provider and Alice is interested in Bob’s data assets. Bob, as a data provider, creates an asset. The data asset should have the following properties:
+
+| ID          | 4                                                                                            |
+|-------------|----------------------------------------------------------------------------------------------|
+| Description | Product EDC Demo Asset 4                                                                     |
+| Type        | HttpData                                                                                     |
+| URL         | [https://jsonplaceholder.typicode.com/todos/4](https://jsonplaceholder.typicode.com/todos/4) |
+
+Action (Bob): Create an asset using the following curl command:
+
+```shell
+curl --location 'http://localhost/bob/management/v2/assets' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+    "@context": {},
+    "asset": {
+        "@type": "Asset",
+        "@id": "4", 
+        "properties": {
+            "description": "Product EDC Demo Asset 4"
+        }
+    },
+    "dataAddress": {
+        "@type": "DataAddress",
+        "type": "HttpData",
+        "baseUrl": "https://jsonplaceholder.typicode.com/todos/4"
+    }
+}'
+```
+
+Now that the asset is created, a policy must be created to define who can access the asset. This time Bob does not want Alice to see and access the asset. So he defines a policy not allowing Alice to access his asset.
+
+Action (Bob): Create the access policy using the following curl command:
+
+```shell
+curl --location 'http://localhost/bob/management/v2/policydefinitions' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "@type": "PolicyDefinitionRequestDto",
+  "@id": "4",
+  "policy": {
+    "@type": "Policy",
+    "odrl:permission": [
+      {
+        "odrl:action": "USE",
+        "odrl:constraint": {
+          "@type": "LogicalConstraint",
+          "odrl:or": [
+            {
+              "@type": "Constraint",
+              "odrl:leftOperand": "BusinessPartnerNumber",
+              "odrl:operator": {
+                "@id": "odrl:eq"
+              },
+              "odrl:rightOperand": "BPNL000000000003"
+            }
+          ]
+        }
+      }
+    ]
+  }
+}' 
+```
+
+Lastly, the asset and the access policy must be combined in a contract definition.
+Action (Bob): Create a contract definition including the asset and the policy you´ve created. For this, use the following curl command:
+
+```shell
+curl --location 'http://localhost/bob/management/v2/contractdefinitions' \
+--header 'Content-Type: application/json' \
+--header 'X-Api-Key: password' \
+--data-raw '{
+    "@context": {},
+    "@id": "4",
+    "@type": "ContractDefinition",
+    "accessPolicyId": "4",
```

Comment:
I would prefer to use different policies here. For example, it would be very illustrative, to have a _restrictive_ access policy, and a _permissive_ contract policy.

Reply:
@jkbquabeck Please add a new section just above this one to create a contract policy. This is the curl:
```
curl --location 'http://localhost/bob/management/v2/policydefinitions' \
--header 'Content-Type: application/json' \
--header 'X-Api-Key: password' \
--data-raw '{
  "@context": {
    "odrl": "http://www.w3.org/ns/odrl/2/"
  },
  "@type": "PolicyDefinitionRequestDto",
  "@id": "5",
  "policy": {
    "@type": "Policy",
    "odrl:permission": [
      {
        "odrl:action": "USE",
        "odrl:constraint": {
          "@type": "LogicalConstraint",
          "odrl:or": [
            {
              "@type": "Constraint",
              "odrl:leftOperand": "BpnCredential",
              "odrl:operator": {
                "@id": "odrl:eq"
              },
              "odrl:rightOperand": "active"
            }
          ]
        }
      }
    ]
  }
}'
```